src/vm/gcenv.ee.h

```diff
@@ -74,6 +87,10 @@ class GCToEEInterface : public IGCToCLR {
     void AnalyzeSurvivorsFinished(int condemnedGeneration);
 
     void VerifySyncTableEntry();
+
+#if defined(__linux__)
```

Comment:
let's not have #ifdef on the interface itself - it's easier to have the interface look the same everywhere; for implementation for windows this is just no op (and you don't need to call it in gc.cpp either).